README.markdown

```diff
@@ -479,11 +479,10 @@ func reticulateSplines(spline: [Double]) -> Bool {
 For functions with long signatures, put each parameter on a new line and add an extra indent on subsequent lines:
 
 ```swift
-func reticulateSplines(
-  spline: [Double], 
-  adjustmentFactor: Double,
-  translateConstant: Int, comment: String
-) -> Bool {
+func reticulateSplines(spline: [Double], 
```

Comment:
when should we move parameters into separate lines? What is "long signature"?

Reply:
we can use https://github.com/realm/SwiftLint/blob/master/Rules.md#multiline-parameters to enforce this, either all parameters on the same line ar all in separate lines with the first parameter being on the same line as `func`